copycat/conceptMapping.py

```diff
@@ -79,6 +79,7 @@ def sameTargetDescriptor(self, other):
     def sameDescriptors(self, other):
         if self.sameInitialDescriptor(other):
             return self.sameTargetDescriptor(other)
+        return False
```

Comment:
Apprecitae your reasoning there.

I've added a ref to your fork to my README, and shall follow your changes with interest.